docs/en/security/posix_acl.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 1
+---
+
+# POSIX ACL
+
+Version 1.2 supports POSIX ACL. For detailed rules, please refer to:
+
+- [POSIX Access Control Lists on Linux](https://www.usenix.org/legacy/publications/library/proceedings/usenix03/tech/freenix03/full_papers/gruenbacher/gruenbacher_html/main.html#:~:text=Access%20Check%20Algorithm&text=The%20ACL%20entries%20are%20looked,matching%20entry%20contains%20sufficient%20permissions.)
+- [setfacl](https://linux.die.net/man/1/setfacl)
+
+## Usage
```

Comment:
Add an example to enable ACL for volume created by old versions.

how to enable it:
1. upgrade all client to 1.2 and remount it
2. juicefs config --enable cli

Reply:
done